pom.xml

```diff
@@ -432,6 +432,35 @@
       </properties>
     </profile>
 
+    <!-- Copy newly built native libraries back to source tree -->
+    <profile>
+      <id>preserve-natives</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>${plugin.antrun.version}</version>
+            <executions>
+              <execution>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target name="Copy compiled native binaries to source-tree">
```

Comment:
Target names -- by convention -- are normally not sentences, per https://ant.apache.org/manual/targets.html